usr/src/cmd/fs.d/smbclnt/fksmbcl/.dbxrc

```diff
@@ -0,0 +1,24 @@
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Nexenta Systems, Inc.  All rights reserved.
+#
+
+set -o emacs
```

Comment:
I think this probably shouldn't be here.  It may override personal preferences of other dbx users...

Reply:
(Even though it is *my* preference)

Reply:
agreed.  just forgot it was there when I committed this.